.gitignore

```diff
@@ -11,6 +11,9 @@ config.js
 *.pdf
 archives
 
+# Cabal config files
+.cabal.yml

```

Comment:
another question is were we should put the conf file, in `~/.cabal/.cabal.yml`?

Reply:
I don't know! Where would you all prefer to have your config file? Right at home or within the ~/.cabal dir ? @noffle @karissa 

I guess it's more tidy to keep it in the dir and the desktop keeps it within it's db dir too.

Reply:
my preference: have the config file live in a config directory for the app (linux is `~/.config/cabal/`), or put it in the `~/.cabal` dir we already use. (and if it's in a dir, it doesn't need to have a `.` prefix)

Reply:
preference for filename: `config.yml`

Reply:
Ok! I just updated it to look for, in this order:
- --config [pathTofile]
- ~/.cabal/config.yml
- ./.cabal.yml